docroot/themes/custom/uids_base/scss/components/grid.scss

```diff
@@ -2,9 +2,20 @@
 @import "uids3/assets/scss/_utilities.scss";
 @import "uids3/components/grid/grid.scss";
 
-.list-container {
+// List container spacing.
+// @todo we would need to add list-container wrapper class to all views, articles list, and taxonomy lists.
```

Comment:
Is this something we want to do or is this the easier solution?

Reply:
I think this has largely been done. This isn't a great place to track this, so let's remove it in favor of checkboxes related to the individual views in the issue summary.

Reply:
FWIW, I previously tried to add relevant checkboxes next to all the lists of cards.